src/index.html

```diff
@@ -3,11 +3,351 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link rel="icon" type="image/png" href="./images/sound_w.png">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
+    <header class="header">
+      <div class="header__content-wrapper">
+        <section class="header__content">
+          <a href="#">
+            <img class="header__logo"
+                src="./images/Boss_img/Bose_logo.png"
+                alt="boss logo"
+            >
+          </a>
+          <div class="header__icon">
+            <a class="header__phone" href="tel:+1-234-555-55-55"></a>
+            <a href="#menu"
+              class="header__menu-toggler"
+            >
+            </a>
+          </div>
+        </section>
+
+        <h1 class="header__title">The world shades.
+          <br>
+          Your music shines.
+        </h1>
+      </div>
+    </header>
+
+    <section class="menu page__menu" id="menu">
+      <div class="menu__content-wrapper">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a href="#">
+              <img class="menu__logo"
+                  src="./images/Boss_img/Bose_logo.png"
+                  alt="bose logo"
+              >
+            </a>
+
+            <a href="#" class="menu_toggler"
+            ></a>
+
+          </div>
+          <nav class="menu__nav">
+            <ul class="menu__list">
+              <li class="menu__item">
+                <a href="#" class="menu__link">home</a>
+              </li>
+              <li class="menu__item">
+                <a href="#" class="menu__link">about us</a>
+              </li>
+              <li class="menu__item">
+                <a href="#" class="menu__link">gallery</a>
+              </li>
+              <li class="menu__item">
+                <a href="#" class="menu__link">testimonials</a>
+              </li>
+              <li class="menu__item">
+                <a href="#" class="menu__link">contacts</a>
```

Comment:
let's make all hrefs to navigate to respective sections

Reply:
Done